### PR TITLE
Clean up travis scripts and add meeting minutes

### PR DESCRIPTION
**Pull request by ericdill**
Checklist
* [ ] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,23 @@
...
+1. `conda env create -f ./.ci_support/environment.yml`
+2. `conda activate conda-forge-docs`
+3. `cd src`
+4. `make html`
+
+## Conda forge dev meetings
+
+A directory containing the minutes from previous dev meetings can be found in /src/minutes or on our website at **insert website link here**.
+These meetings occur every two weeks on Wednesday from 17:30-18:00 UTC.
+A link to the google calendar item can be found [here](https://calendar.google.com/event?action=TEMPLATE&tmeid=dGozZTc1NjhvcmY1cTh1Nm85ODJhZXBibWNfMjAxOTA3MTFUMTUwMDAwWiBlZGlsbEBhbmFjb25kYS5jb20&tmsrc=edill%40anaconda.com).
```
Comment (xhochy) [inline README.md]: The link to the event doesn't work for me.

File: README.md [hunk 2]
```diff
@@ -2,3 +2,23 @@
...
+
+The docs are built on travis ci with the `.travis.yml` file at the root of this repository.
+`.travis.yml` uses [conda-execute](https://github.com/conda-tools/conda-execute) to run the `.ci_scripts/update_docs` script.
+To build the docs locally, you should do the following:
+1. `conda env create -f ./.ci_support/environment.yml`
+2. `conda activate conda-forge-docs`
+3. `cd src`
+4. `make html`
+
+## Conda forge dev meetings
```
Comment (croth1) [inline README.md]: I suggest `Conda forge` ->  `conda-forge` for consistency.

File: .travis.yml [hunk 3]
```diff
@@ -29,12 +29,9 @@ install:
     - conda config --set show_channel_urls true
     - conda config --set auto_update_conda false
 
-    # Now do the things we need to do to install it.
-    - conda install conda-execute --yes --quiet -c conda-forge
-
 script:
     - echo "Updating docs";
-    - source ./.ci_scripts/update_docs;
+    - bash ./ci_scripts/update_docs;
```
Comment (scopatz) [inline .travis.yml]: Should this be `bash ./.ci_scripts/update_docs;`?

File: README.md (hunk 1 above)
Comment (scopatz) [reply README.md]: Does this link work? `https://calendar.google.com/event?action=TEMPLATE&tmeid=Z2lraDk2a205cGUxdDkxYmNybXQxMGIxMGtfMjAxOTA3MjRUMTcwMDAwWiBzY29wYXR6QG0&tmsrc=scopatz%40gmail.com&scp=ALL`

**Comment by scopatz** [comment]
Couple of small comments from me :wink:

File: README.md (hunk 1 above)
Comment (ericdill) [reply README.md]: That one works for me. Thanks @scopatz. Thanks for checking the link @xhochy !

File: .travis.yml (hunk 3 above)
Comment (ericdill) [reply .travis.yml]: yes 😞

File: README.md (hunk 1 above)
Comment (xhochy) [reply README.md]: @scopatz  Also works for me but the time in the link doesn't match the text. This Meeting is shown to me as 18:00-18:30 CEST (which is then 16:00-16:30 UTC).

**Comment by scopatz** [comment]
Thanks @ericdill!

**Pull request merged**